frontend/src/features/manageVehicles/components/list/List.tsx

```diff
@@ -193,7 +197,7 @@ export const List = memo(
             query.refetch();
           }
         } else {
-          handleError();
+          handleError(response.headers["x-correlation-id"]);
         }
       } catch {
         handleError();
```

Comment:
Would we need to pass the correlation id header here as well? and in the handleError call on line 184?

Reply:
I've corrected line 184 to include the correlationId. This catch block, I'm not sure if it will ever happen. I don't think it needs a try-catch block but that might be a ticket for later to clean up. In the meantime, it still fails gracefully, just without a correlation id.

Reply:
ok sounds good